profiler/src/main/java/com/splunk/opentelemetry/profiler/snapshot/Volume.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Splunk Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.splunk.opentelemetry.profiler.snapshot;
+
+public enum Volume {
+  OFF,
+  HIGHEST;
+
+  static Volume fromString(String value) {
+    if (value == null) {
+      return OFF;
+    }
+
+    try {
+      return Volume.valueOf(value.toUpperCase());
```

Comment:
We usually use `toUpperCase` and `toLowerCase` with locale like `toUpperCase(Locale.ROOT)`. Using explicit locale helps against surprises where in the user locale characters are converted in an unexpected way. For example in Turkish language the upper case of `i` is not `I` but `İ` which here could result in a difficult to debug issue.